Code-review/index.html

```diff
@@ -0,0 +1,197 @@
+        </div>
+        <div class="middle">
+            <li>
+                <input type="text" placeholder="seacrh"/>
+            </li>
+            <li>
+                <i class="fa-sharp fa-solid fa-magnifying-glass"></i>
+            </li>
+            <li class="micro"><i class="fa-solid fa-microphone"></i></li>
+          
+        </div>
+        <div class ="right" >
+            <i class="fa-solid fa-video"></i>
+            
+                <i class="fa-regular fa-bell"><span>9+</span></i>
+        
+            
+            <i class="fa-solid fa-user-plus"></i>
+            
+           
+        </div>
+    </nav>
+    <div class="main">
+    <section class="video">
+     
+        <iframe width="820" height="520" src="https://www.youtube.com/embed/V74l_zS1x8E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        <p style="font-size:18px; font-weight:1000 ;font-style:capitalize;">Contributing to Open Source Part 1:The Easy Way</p>
+        <ul class="list">
+            <li>68220 views.</li>
+            <li>Feb 28 2014</li>
+            <li><i class="fa-sharp fa-solid fa-thumbs-up" style="margin-left:50px ;"><span style="margin-left: 5px;">488</span></i></li>
+            <li><i class="fa-solid fa-thumbs-down"><span style="margin-left: 5px;"> DISLIKE</span></i></li>
+            <li><i class="fa-sharp fa-solid fa-share"><span style="margin-left: 5px;">SHARE</span></i></li>
+            <li><i class="fa-solid fa-arrow-down"><span style="margin-left: 5px;">DOWNLOAD</span></i></li>
+            <li><i class="fa-solid fa-scissors"> <span style="margin-left: 5px;">CLIP</span></i></li>
+            <li><i class="fa-solid fa-square-plus"> <span style="margin-left: 5px;">SAVE</span></i></li>
+            <li style="margin-bottom: 20px;">...</li>
+        </ul>
+        
+    </section>
+
+    <section class="thumbnail">
+            <li><i class="fa-sharp fa-solid fa-chevron-left"></i></li>
+         <li> <button class="btn">program</button></li> 
+         
+         <li> <button class="btn">listenable</button></li>
+         <li>
+            <button class="btn">watched</button>
+         </li>
+         <li>
+            <i class="fa-sharp fa-solid fa-chevron-right"></i>
+         </li>
+         <div class="thumbimages">
+            <img class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
```

Comment:
Please i would kindly suggest that you try to avoid using inline styling as this makes the code look unclean. it would be better if you give them classes and do the styling from the CSS file.😄
Please do this for all the cases in the file.